capstone-app/src/index.html

```diff
@@ -4,6 +4,7 @@
   <meta charset="utf-8">
   <title>CapstoneApp</title>
   <base href="/">
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDwcRTXYZaM4tOayy4Jvw5h79sU-Gdh2_0&libraries=places"></script>
```

Comment:
I think this is your personal API key. You should use one associated with the GMapr-STEP project ID. If that isn't ready yet, put a TODO here that it needs to be changed. In fact, you may want to remove the API key from the Github repo entirely so that other people can't use the key and rack up requests.

```
<!-- TODO(smontana): Replace API key when available -->
```